src/main/java/in/twizmwaz/cardinal/command/WhitelistCommands.java

```diff
@@ -151,21 +151,17 @@ public static void kick(final CommandContext args, final CommandSender sender) t
         ChatUtil.getGlobalChannel().sendLocalizedMessage(new UnlocalizedChatMessage(ChatColor.RED + "{0}", ChatConstant.GENERIC_KICKED_NOT_WHITELISTED.asMessage()));
     }
 
-    @Command(aliases = {"team"}, desc = "Adds everyone on a team to the whitelist.", max = 1, min = 1)
+    @Command(aliases = {"team"}, desc = "Adds everyone on a team to the whitelist.", min = 1)
     @CommandPermissions("cardinal.whitelist.team")
     public static void team(final CommandContext args, final CommandSender sender) throws CommandException {
         int count = 0;
-        String msg = "";
-        for (int i = 2; i < args.argsLength(); i++) {
```

Comment:
@dentmaged as i said  b4, if there is a team called "Team Red" and a team called "Team Blue", /wl team team should wl both, not pick the first one (alphabetically) and wl them